tests/robot-cases/Group1-Nightly/Common.robot

```diff
@@ -556,6 +556,38 @@ Test Case - Tag CRUD
     Should Contain Tag   456
     Delete A Tag   latest
     Should Not Contain Tag   latest
+    Close Browser
+
+Test Case - GC Untagged Images
+    Init Chrome Driver
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    ${d}=    Get Current Date    result_format=%m%s
+    Create An New Project  project${d}
+    Push Image With Tag  ${ip}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}  project${d}  hello-world  latest
+    # make hello-world untagged
+    Go Into Project   project${d}
+    Go Into Repo   hello-world
+    Go Into Artifact   latest
+    Should Contain Tag   latest
+    Delete A Tag   latest
+    # run gc without param delete untagged artifacts checked,  should not delete hello-world:latest

```

Comment:
Should Not Contain Tag   latest
Please add this after delete a tag.

Reply:
Fixed